crates/astria-conductor/src/executor.rs

```diff
@@ -327,247 +305,243 @@ impl<C: ExecutionClient> Executor<C> {
-        // should be empty because 1 block was executed and finalized, which deletes it from the map
-        assert!(executor.sequencer_hash_to_execution_hash.is_empty());
-    }
-
-    #[tokio::test]
-    async fn skip_unexecuted_da_block_with_no_transactions() {
-        let mut conf = get_test_config();
-        let chain_id = ChainId::new(conf.chain_id.as_bytes().to_vec()).unwrap();
-        conf.disable_empty_block_execution = true;
-        let finalized_blocks = Arc::new(Mutex::new(HashSet::new()));
-        let execution_client = MockExecutionClient {
-            finalized_blocks: finalized_blocks.clone(),
-        };
-        let (mut executor, _) = Executor::new(
-            execution_client,
-            chain_id,
-            conf.disable_empty_block_execution,
-        )
-        .await
-        .unwrap();
-
-        let block: SequencerBlockSubset = get_test_block_subset();
-        let previous_execution_state = executor.execution_state.clone();
-
-        executor
-            .handle_block_received_from_data_availability(block)
-            .await
-            .unwrap();
-
-        // should not have executed or finalized the block
-        assert!(finalized_blocks.lock().await.is_empty());
-        assert!(
-            finalized_blocks
-                .lock()
-                .await
-                .get(&executor.execution_state)
-                .is_none()
-        );
-        assert_eq!(previous_execution_state, executor.execution_state,);
-        // should be empty because nothing was executed
-        assert!(executor.sequencer_hash_to_execution_hash.is_empty());
-    }
-
-    #[tokio::test]
-    async fn execute_unexecuted_da_block_with_no_transactions() {
-        let conf = get_test_config();
-        let chain_id = ChainId::new(conf.chain_id.as_bytes().to_vec()).unwrap();
-        let finalized_blocks = Arc::new(Mutex::new(HashSet::new()));
-        let execution_client = MockExecutionClient {
-            finalized_blocks: finalized_blocks.clone(),
-        };
-        let (mut executor, _) = Executor::new(
-            execution_client,
-            chain_id,
-            conf.disable_empty_block_execution,
-        )
-        .await
-        .unwrap();
-
-        let block: SequencerBlockSubset = get_test_block_subset();
-        let expected_execution_state = hash(&executor.execution_state);
-
-        executor
-            .handle_block_received_from_data_availability(block)
-            .await
-            .unwrap();
-
-        assert!(
-            finalized_blocks
-                .lock()
-                .await
-                .get(&executor.execution_state)
-                .is_some()
-        );
-        assert_eq!(expected_execution_state, executor.execution_state);
-        // should be empty because block was executed and finalized, which deletes it from the map
-        assert!(executor.sequencer_hash_to_execution_hash.is_empty());
-    }
-}
+// #[cfg(test)]
```

Comment:
a new issue should be created to fix and add these tests back in

Reply:
addressed by @sambukowski's commit